### ISSUE TITLE
[Feature request] Hit not-tabs area to new a note

### ISSUE THREAD

**Issue by emisjerry** [Quorafind/Obsidian-Double-Click-Tab#10]
If pressed at non-tabs area, it can new a note in a new tab.

Some theme moves the + icon to the rightmost position, it'll be better if we can double-click on the non-tabs area to new a note.

**Comment by Quorafind** [comment]
I don't think it is possible to do it unlsess hack obsidian's default behavior because now when you double click on none-tabs area, obsidian would maximize the window or return.

**Comment by emisjerry** [comment]
Double click at Obsidian window title bar will maxmize the app. But tabs row won't do anything now.

![image](https://user-images.githubusercontent.com/131511/198499606-1dc265d8-f65b-4f66-9794-8534bfab869a.png)
